pkg/controllers/dynakube/extension/eec/statefulset.go

```diff
@@ -273,11 +272,33 @@ func buildContainerVolumeMounts(dk *dynakube.DynaKube) []corev1.VolumeMount {
 		})
 	}
 
+	if dk.Spec.Templates.ExtensionExecutionController.TlsRefName != "" {
+		volumeMounts = append(volumeMounts, corev1.VolumeMount{
+			Name:      consts.ExtensionsCustomTlsCertificate,
+			MountPath: baseSecretsPath + "/eec",
```

Comment:
Why two different mountpoints are used? Eventually it's secret created by operator or user. Just different name of the secret.

Reply:
I agree. The mount point should always be the same, whether the cert is created by the user or by the operator.

Reply:
fixed in bc76d466